### PR TITLE
move dynamic compute cluster ip address fix explanation to release notes, add parallelcluster CLI installation link

### PR DESCRIPTION
**Pull request by kct22aws**
move dynamic compute cluster ip address fix explanation to release notes, add parallelcluster CLI installation link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ A key pair is needed for access to the head node of the cluster. You may use an
 
 ### AWS ParallelCluster Python package
 
-AWS ParallelCluster Python package is needed in a local environment (i.e., your Mac/PC desktop with a CLI terminal or an AWS Cloud9) where you issue the command to launch the creation process for your HPC environment in AWS.
+AWS ParallelCluster Python package is needed in a local environment (i.e., your Mac/PC desktop with a CLI terminal or an AWS Cloud9) where you issue the command to launch the creation process for your HPC environment in AWS. See [here](https://docs.aws.amazon.com/parallelcluster/latest/ug/install-v3-virtual-environment.html) for instructions about installing AWS ParallelCluster Python package in your local node.
```
Comment (jeffhataws) [inline README.md:25]: "local environment" instead of "local node".

Comment (kct22aws) [reply README.md]: Done in new commit

Comment (jeffhataws) [reply README.md:25]: Still shows "local node" in the last sentence.

**Review by jeffhataws — APPROVED** [review]

File: releasenotes.md [hunk 2]
```diff
@@ -23,4 +23,30 @@ it is because one of the hostnames in `/etc/hosts` is longer than 63 characters.
 srun -N <number of nodes> sudo sed -i 's/\([0-9]\) .*pcluster / /' /etc/hosts
 ```
 
+* Relaunch a dynamic cluster created with `MinCount = 0` may fail due to compute nodes IP address mismatch.
 
+For dynamic cluster with `MinCount = 0`, /etc/hosts IP addresses of compute nodes may not match with what's in `nslookup` upon cluster relaunch. Therefore, for your information, a temporary workaround is included in `install_neuron.sh` post-install script:
```
Comment (awsjoshir) [inline releasenotes.md:28]: based on what I saw in the pipeline tests even without relaunch we noticed that this workaround was needed. the pipeline tests launch dynamic cluster and run jobs which then launches the workers and once job completes, we tear it down. It fails this use case without the workaround.

File: releasenotes.md [hunk 3]
```diff
@@ -23,4 +23,30 @@ it is because one of the hostnames in `/etc/hosts` is longer than 63 characters.
...
+
+
+```
+
+   INFO: Inconsistency detected by ld.so: ../elf/dl-tls.c: 488: _dl_allocate_tls_init: Assertion `listp->slotinfo[cnt].gen <= GL(dl_tls_generation)' failed!
+   INFO: 2022-10-03 02:16:04.488054: W tensorflow/core/distributed_runtime/rpc/grpc_remote_master.cc:157] RPC failed with status = "UNAVAILABLE: Connection reset by peer" and grpc_error_string = "{"created":"@1664763364.487962663","description":"Error received from peer ipv4:10.0.9.150:41677","file":"external/com_github_grpc_grpc/src/core/lib/surface/call.cc","file_line":1056,"grpc_message":"Connection reset by peer","grpc_status":14}", maybe retrying the RPC
+
+```
+This error may occur intermittently when using GNU C Library glibc 2.26. To find out what version you have, run ```ldd --version```. glibc 2.27 provides a workaround and therefore the error is fixed in Ubuntu20. For more information on this issue, see [Neuron troubleshooting guide](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/frameworks/torch/torch-neuronx/training-troubleshooting.html#error-assertion-listp-slotinfo-cnt-gen-gl-dl-tls-generation-failed-followed-by-rpc-failed-with-status-unavailable-connection-reset-by-peer).
```
Comment (awsjoshir) [inline releasenotes.md:48]: Is this error expected on U20 and needs to be patched or can happen on both AL2 and U20, both needing to be patched? Can you please clarify ?

Comment (kct22aws) [reply releasenotes.md:48]: ON u20 it works. Not confirmed on AL2 yet.

File: releasenotes.md (hunk 2 above)
Comment (kct22aws) [reply releasenotes.md:28]: ack

**Review by awsjoshir — APPROVED** [review]
Thanks KC!

**Review by aws-maens — APPROVED** [review]

**Pull request merged**